expertise/create_dataset.py

```diff
@@ -36,12 +36,31 @@ def convert_to_list(self, config_invitations):
         assert isinstance(invitations, list), 'Input should be a str or a list'
         return invitations
 
+    def get_paper_notes(self, author_id, dataset_params):
+
+        use_bids_as_expertise = dataset_params.get('bid_as_expertise', False)
+
+        if use_bids_as_expertise:
+            bid_invitation = dataset_params['bid_invitation']
+            bids = openreview.tools.iterget_edges(self.openreview_client, invitation=bid_invitation, tail=author_id)
+            note_ids = [e.head for e in bids if e.label in ['Very High', 'High']]
+
+            notes = self.openreview_client.get_notes_by_ids(ids=note_ids)
+            difference = list(set(note_ids) - set([n.id for n in notes]))
+            if difference:
+                print('difference', difference)
```

Comment:
Is this difference to check if there are papers that were desk rejected after the reviewer bid on them?

Reply:
Ohhh you are right, I should check the invitation id of each note!